packages/model-viewer/src/model-viewer-base.ts

```diff
@@ -13,18 +13,19 @@
  * limitations under the License.
  */
 
-import {ReactiveElement} from 'lit';
-import {property} from 'lit/decorators.js';
-import {Camera as ThreeCamera, Event as ThreeEvent, Vector2, Vector3, WebGLRenderer} from 'three';
-
-import {HAS_INTERSECTION_OBSERVER, HAS_RESIZE_OBSERVER} from './constants.js';
-import {$updateEnvironment} from './features/environment.js';
-import {makeTemplate} from './template.js';
-import {$evictionPolicy, CachingGLTFLoader} from './three-components/CachingGLTFLoader.js';
-import {ModelScene} from './three-components/ModelScene.js';
-import {ContextLostEvent, Renderer} from './three-components/Renderer.js';
-import {clamp, debounce} from './utilities.js';
-import {ProgressTracker} from './utilities/progress-tracker.js';
+import { ReactiveElement } from 'lit';
```

Comment:
Looks like your formatter isn't set up to use our .clang-format style. Can you fix that to clean up this diff?

Reply:
had to do it manually, somehow vscode doesn't pick that for me:
./node_modules/.bin/clang-format -i packages/model-viewer/src/model-viewer-base.ts